x-pack/plugins/security_solution/cypress/integration/timeline_local_storage.spec.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { loginAndWaitForPage } from '../tasks/login';
+import { HOSTS_PAGE } from '../urls/navigation';
+import { openEvents } from '../tasks/hosts/main';
+import { DRAGGABLE_HEADER } from '../screens/timeline';
+import { TABLE_COLUMN_EVENTS_MESSAGE } from '../screens/hosts/external_events';
+import { waitsForEventsToBeLoaded, openEventsViewerFieldsBrowser } from '../tasks/hosts/events';
+import { removeColumn, resetFields } from '../tasks/timeline';
+
+describe('persistent timeline', () => {
+  before(() => {
+    loginAndWaitForPage(HOSTS_PAGE);
+    openEvents();
+    waitsForEventsToBeLoaded();
+  });
+
+  afterEach(() => {
+    openEventsViewerFieldsBrowser();
+    resetFields();
+  });
+
+  it('persist the deletion of a column', () => {
+    cy.get(DRAGGABLE_HEADER)
+      .eq(TABLE_COLUMN_EVENTS_MESSAGE)
+      .invoke('text')
+      .should('equal', 'message');
+    removeColumn(TABLE_COLUMN_EVENTS_MESSAGE);
+    cy.reload();
```

Comment:
After the reload method we need to add first:  `cy.contains('a', 'Security')` and then `waitsForEventsToBeLoaded ()`. With the first method we are first waiting for the page to be loaded and then we are waiting for the events table to be ready as well. Adding those lines of code we are minimising the risk of this test to be flaky.